<?xml version="1.0" encoding="utf-8"?>
<launch>
	<arg name="veh" default="$(env VEHICLE_NAME)"/>
	<arg name="pkg_name" default="pure_pursuit_lfv" doc="name of the package"/>
	<arg name="node_name" default="pure_pursuit_controller_node" doc="name of the node"/>
	<arg name="vehicle_avoidance" default="true"/>
	<arg name="verbose" default="false"/>

	<group ns="$(arg veh)">
		<node pkg="$(arg pkg_name)" type="$(arg node_name).py" name="$(arg node_name)" output="screen" required="true">
			<param name="vehicle_avoidance" value="$(arg vehicle_avoidance)"/>
			<param name="verbose" value="$(arg verbose)"/>
		</node>
	</group>
</launch>
